components/accordion/index.css

```diff
@@ -11,17 +11,295 @@ governing permissions and limitations under the License.
+	/* Focus indicator */
+	--spectrum-accordion-focus-indicator-color: var(
+		--spectrum-focus-indicator-color
+	);
+	/* Divider */
+	--spectrum-accordion-divider-color: var(--spectrum-gray-300);
+	--spectrum-accordion-min-block-size: max(
+		var(--mod-accordion-item-height, var(--spectrum-accordion-item-height)),
+		calc(
+			var(
+					--mod-accordion-item-header-top-to-text-space,
+					var(--spectrum-accordion-item-header-top-to-text-space)
+				) +
+				var(
+					--mod-accordion-item-header-bottom-to-text-space,
+					var(--spectrum-accordion-item-header-bottom-to-text-space)
+				) +
+				(
+					var(
+							--mod-accordion-item-header-font-size,
+							var(--spectrum-accordion-item-header-font-size)
+						) *
+						var(
+							--mod-accordion-item-header-line-height,
+							var(--spectrum-accordion-item-header-line-height)
+						)
+				)
+		)
+	);
+}
+
+.spectrum-Accordion--compact {
+	&.spectrum-Accordion--sizeS {
+		--spectrum-accordion-item-height: var(--spectrum-component-height-75);
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-compact-small
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-compact-small
+		);
+	}
+
+	&.spectrum-Accordion--sizeM {
+		--spectrum-accordion-item-height: var(--spectrum-component-height-100);
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-compact-medium
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-compact-medium
+		);
+	}
+
+	&.spectrum-Accordion--sizeL {
+		--spectrum-accordion-item-height: var(--spectrum-component-height-200);
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-compact-large
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-compact-large
+		);
+	}
+
+	&.spectrum-Accordion--sizeXL {
+		--spectrum-accordion-item-height: var(--spectrum-component-height-300);
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-compact-extra-large
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-compact-extra-large
+		);
+	}
+}
+
+.spectrum-Accordion--spacious {
+	&.spectrum-Accordion--sizeS {
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-small-top-to-text-spacious
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-spacious-small
+		);
+	}
+
+	&.spectrum-Accordion--sizeM {
+		--spectrum-accordion-item-header-line-height: 1.278;
+		--spectrum-accordion-item-header-top-to-text-space: var(
+			--spectrum-accordion-top-to-text-spacious-medium
+		);
+		--spectrum-accordion-item-header-bottom-to-text-space: var(
+			--spectrum-accordion-bottom-to-text-spacious-medium
+		);
```

Comment:
I would move these under `.spectrum-Accordion--spacious`, to be the default for this density.